content/en/docs/components/katib/hyperparameter.md

```diff
@@ -0,0 +1,436 @@
+
+If you installed Katib as part of Kubeflow, you can access the
+Katib UI from the Kubeflow UI:
+
+1. Open the Kubeflow UI. Check the guide to
+   [accessing the central dashboard](/docs/components/central-dash/overview/).
+1. Click **Katib** in the left-hand menu.
+
+Alternatively, you can set port-forwarding for the Katib UI service:
+
+```shell
+kubectl port-forward svc/katib-ui -n kubeflow 8080:80
+```
+
+Then you can access the Katib UI at this URL:
+
+```shell
+http://localhost:8080/katib/
+```
+
+Check [this guide](https://github.com/kubeflow/katib/tree/master/pkg/ui/v1beta1)
+if you want to contribute to Katib UI.
+
+## Examples
+
+This section introduces some examples that you can run to try Katib.
+
+<a id="random-algorithm"></a>
+
+### Example using random algorithm
+
+You can create an experiment for Katib by defining the experiment in a YAML
+configuration file. The YAML file defines the configurations for the experiment,
+including the hyperparameter feasible space, optimization parameter,
+optimization goal, suggestion algorithm, and so on.
+
+This example uses the [YAML file for the
+random algorithm example](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/random-example.yaml).
+
+The random algorithm example uses an MXNet neural network to train an image
+classification model using the MNIST dataset. You can check training container source code
+[here](https://github.com/kubeflow/katib/tree/master/examples/v1beta1/mxnet-mnist).
+The experiment runs twelve training jobs with various hyperparameters and saves the results.
+
+If you installed Katib as part of Kubeflow, you can't run experiments in Kubeflow namespace.
+Run the following commands to change namespace and launch an experiment using the random algorithm
+example:
+
+1. Download the example:
+
+   ```shell
+   curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/random-example.yaml --output random-example.yaml
+   ```
+
+1. Edit `random-example.yaml` and change the following line to use your Kubeflow user profile namespace:
+
+   ```shell
+   Namespace: kubeflow
+   ```
+
+1. Deploy the example:
+
+   ```shell
+   kubectl apply -f random-example.yaml
+   ```
+
+This example embeds the hyperparameters as arguments. You can embed
+hyperparameters in another way (for example, using environment variables)
+by using the template defined in the `trialTemplate.trialSpec` section of
+the YAML file. The template uses the
+[unstructured](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured)
+format and substitutes parameters from the `trialTemplate.trialParameters`.
+Follow the [trial template guide](/docs/components/katib/trial-template/)
+to know more about it.
+
+This example randomly generates the following hyperparameters:
+
+- `--lr`: Learning rate. Type: double.
+- `--num-layers`: Number of layers in the neural network. Type: integer.
+- `--optimizer`: Optimizer. Type: categorical.
```

Comment:
Would it make sense to say "A gradient descent optimizer for neural network training"? So many variables are being optimized during training, especially in AutoML, it may get confusing for a few:

```suggestion
- `--lr`: Learning rate. Type: double.
- `--num-layers`: Number of layers in the neural network. Type: integer.
- `--optimizer`: Gradient descent optimizer for neural network training. Type: categorical.
```

Reply:
I think we can re-name it to: "Optimization method to change the neural network attributes".
WDYT ?

Reply:
If Katib (AutoML) is for classic deep learning stuff only right now, then it's trying to update the weights and minimize the loss function using the optimizer (with the `lr` using gradient descent, until there's a better way to optimize).

One of the examples on `kubeflow/katib` says it can be stoch. gradient descent/SGD or Adam, so I think we have to be precise here:

"`--optimizer`: Gradient descent optimizer for neural network training, implements backpropagation. Type: categorical."

Or something along those lines.